stdlib/logging/config.pyi

```diff
@@ -15,9 +16,63 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_TKey = TypeVar("_TKey", bound=str)
+_TValue = TypeVar("_TValue")
+
 DEFAULT_LOGGING_CONFIG_PORT: int
 RESET_ERROR: int  # undocumented
 IDENTIFIER: Pattern[str]  # undocumented
+_listener: Optional[Any]
+dictConfigClass = DictConfigurator
+
+def _resolve(name: _TKey) -> _TKey: ...
```

Comment:
Our convention is to not add `_non_public` members, unless you have a specific need for a specific `_non_public` member. Seems like you deleted these already, but somehow they got added back?